docs/ARCHITECTURE.md

```diff
@@ -139,6 +206,21 @@ $text = AiClient::generateTextResult(
 
 _Note: Unlike the previous two examples, to require JSON output it is necessary to go the verbose route, since it is impossible for the SDK to detect whether you require JSON output purely from the prompt input. Therefore this code example contains the logic to manually search for suitable models and then use one of them for the task._
 
+##### Fluent API
+```php
+// Verbose.
+$text = AiClient::prompt('Transform the following CSV content into a JSON array of row data.')
+    ->asJsonResponse()
+    ->usingOutputSchema(['name' => 'string', 'age' => 'integer'])
+    ->generateText();
+
+// Simple.
+$text = AiClient::prompt('Transform the following CSV content into a JSON array of row data.')
+    ->asJsonResponse(['name' => 'string', 'age' => 'integer'])
+    ->generateText();
```

Comment:
IMO, we can skip this. Maybe a bit too much nuance for this list of examples to clarify here that you could also omit the `asJsonResponse()`. To be clear, I agree this should be possible, but not worth including here.

Also, if we kept this, this would also need to be updated to use proper JSON schema, in which case the `Simple` annotation doesn't feel entirely right anymore :)